client/package.json

```diff
@@ -9,9 +9,10 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@date-io/dayjs": "^1.3.8",
-    "@material-ui/core": "^4.1.3",
-    "@material-ui/pickers": "^3.2.0",
+    "@date-io/core": "^1.3.6",
+    "@date-io/dayjs": "^1.3.9",
+    "@material-ui/core": "^4.0.0",
```

Comment:
you are downgrading this package, remember to always pull.

Reply:
Ok